core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,38 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var suppressAnswer = responsesService.getAnswerGroups();
+        var defaultAnswers;
+        // For choice length, have indices 0 through choice length - 1
+        for (var x = 0; x < choices; x++) {
+          if (suppressAnswer['' + x + ''] === undefined) {
+            // Return false if no object exists for the index
+            /* Note that multiple choice only allows for
```

Comment:
Hey @seanlip The only way I could think to do this was to create arrays to store the index values for the choices and the answerGroups and compare the two. It seems to be working for all of the cases, but is that too clunky?